### DISCUSSION TITLE
Ghost score mod proposal

### DISCUSSION THREAD

**Discussion by Joppe27** [ppy/osu#13119] [Ideas] [answered]
Hi, I think it would be fun if there would be a mod where you can compete against your best score on that map kind of like ghost races in mario kart. You can see if you're doing better or worse than your best while you're playing in realtime. This might be a little useless for standard and taiko because of the huge differences in score when combobreaking but it would definitely be fun in mania!
It would be like multiplayer on stable but instead of competing against other players you try to beat your own score.
This is only an idea, so if you're not up for it that's completely fine! Thanks :)

**Comment by peppy** [comment, marked as answer]
"difference in score" is the same across all rulesets.

this is planned for leaderboards in general, not just as a mod

**Comment by Joppe27** [comment]
> "difference in score" is the same across all rulesets.
> 
> this is planned for leaderboards in general, not just as a mod

Oh, that's great! Can I close this issue then?

**Comment by peppy** [comment]
Leaving open is fine for now.
